client/users/mini-match-history.tsx

```diff
@@ -168,20 +175,20 @@ export function ConnectedGameListEntry({
     return 'unknown'
   }, [results, forUserId])
 
-  const matchType = getGameTypeLabel(game)
-  const mapName = map?.name ?? 'Unknown map'
+  const matchType = getGameTypeLabel(game, t)
+  const mapName = map?.name ?? t('games.miniMatchHistory.mapNameUnknown', 'Unknown map')
```

Comment:
These sorts of strings are not really particular to any UI context and should be pulled from some common place